src/ResourceManager/Sql/Commands.Sql/Database Backup/Cmdlet/RemoveAzureRmSqlDatabaseLongTermRetentionBackup.cs

```diff
@@ -0,0 +1,201 @@
+    {
+        /// <summary>
+        /// Parameter set name for the default remove.
+        /// </summary>
+        private const string RemoveBackupDefaultSet = "RemoveBackupDefault";
+
+        /// <summary>
+        /// Parameter set name for remove with an input object.
+        /// </summary>
+        private const string RemoveBackupByInputObjectSet = "RemoveBackupByInputObject";
+
+        /// <summary>
+        /// Parameter set name for remove with a resource ID.
+        /// </summary>
+        private const string RemoveBackupByResourceIdSet = "RemoveBackupByResourceId";
+
+        /// <summary>
+        /// Gets or sets the name of the location the backup is in.
+        /// </summary>
+        [Parameter(Mandatory = true,
+            ParameterSetName = RemoveBackupDefaultSet,
+            ValueFromPipelineByPropertyName = true,
+            Position = 0,
+            HelpMessage = "The location of the backups' source server.")]
+        [ValidateNotNullOrEmpty]
+        [LocationCompleter("Microsoft.Sql/locations/longTermRetentionServers")]
+        public virtual string Location { get; set; }
+
+        /// <summary>
+        /// Gets or sets the name of the server.
+        /// </summary>
+        [Parameter(Mandatory = true,
+            ParameterSetName = RemoveBackupDefaultSet,
+            ValueFromPipelineByPropertyName = true,
+            Position = 1,
+            HelpMessage = "The name of the Azure SQL Server the backup is under.")]
+        [ValidateNotNullOrEmpty]
+        public string ServerName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the name of the database.
+        /// </summary>
+        [Parameter(Mandatory = true,
+            ParameterSetName = RemoveBackupDefaultSet,
+            ValueFromPipelineByPropertyName = true,
+            Position = 2,
+            HelpMessage = "The name of the Azure SQL Database the backup is from.")]
+        [ValidateNotNullOrEmpty]
+        public string DatabaseName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the backup name.
+        /// </summary>
+        [Parameter(Mandatory = true,
+            ParameterSetName = RemoveBackupDefaultSet,
+            ValueFromPipelineByPropertyName = true,
+            Position = 3,
+            HelpMessage = "The name of the backup.")]
+        [ValidateNotNullOrEmpty]
+        public string BackupName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the LTR Backup object to remove.
+        /// </summary>
+        [Parameter(ParameterSetName = RemoveBackupByInputObjectSet,
+            Mandatory = true,
+            Position = 0,
+            ValueFromPipeline = true,
+            HelpMessage = "The Database Long Term Retention Backup object to remove.")]
+        [ValidateNotNullOrEmpty]
+        public AzureSqlDatabaseLongTermRetentionBackupModel InputObject { get; set; }
```

Comment:
move InputObject and ResourceId above BackupName - makes more sense in help file (be sure to regenerate file).

Reply:
Done.